### ISSUE TITLE
inference speed comparison  yolo11 vs yolo26

### ISSUE THREAD

**Issue by mountain111** [ultralytics/ultralytics#24276]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

<img width="1064" height="433" alt="Image" src="https://github.com/user-attachments/assets/4b774bae-2cca-4639-b1a4-75f60e0adfac" />

<img width="439" height="197" alt="Image" src="https://github.com/user-attachments/assets/62064230-269b-4137-9d31-a3260dd9c519" />

According to the reported inference speed, yolo26 does not pose advantages over yolo11. Is it true?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @mountain111, thank you for your interest in Ultralytics 🚀! This is an automated response to help get your question routed quickly, and an Ultralytics engineer will also assist you soon.
We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.
Since this is a performance comparison ❓ question, please share as much information as possible, including your exact model variants, input image size, hardware, runtime backend, export format if any, batch size, and any benchmarking logs or scripts you used. A small reproducible benchmark is especially helpful 🙏
If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.
If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).
Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.
## Upgrade
Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:
```bash
pip install -U ultralytics
```
## Environments
YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):
- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>
## Status
<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>
If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
YOLO26 doesn't need NMS, so it makes up the speed difference in postprocessing speed.

The table only shows inference speed. It doesn't include postprocessing speed.

**Comment by mountain111** [comment]
Got it.Thanks for your reply.-------- 原始邮件 --------发件人： Mohammed Yasin ***@***.***>日期： 2026年4月20日周一 下午7:37收件人： ultralytics/ultralytics ***@***.***>抄送： mountain111 ***@***.***>, Mention ***@***.***>主    题： Re: [ultralytics/ultralytics] inference speed comparison  yolo11 vs yolo26 (Issue #24276)Y-T-G left a comment (ultralytics/ultralytics#24276)
YOLO26 doesn't need NMS, so it makes up the speed difference in postprocessing speed.
The table only shows inference speed. It doesn't include postprocessing speed.

**Comment by glenn-jocher** [comment]
You're welcome — if you benchmark locally, compare end-to-end latency rather than forward pass alone, since YOLO26 removes NMS; the [YOLO11 vs YOLO26 comparison](https://docs.ultralytics.com/compare/yolo11-vs-yolo26/) and [end-to-end detection guide](https://docs.ultralytics.com/guides/end2end-detection/) cover that in more detail.
